cmd/destroy/destroy.go

```diff
@@ -539,15 +539,27 @@ func (pc *destroyCommand) waitForNamespaceDestroyAllToComplete(ctx context.Conte
 			case "Active":
 				if hasBeenDestroyingAll {
 					// when status is active again check if all resources have been correctly destroyed
+					// list configmaps that belong okteto deployments
+					resourcesLabels := map[string]bool{
```

Comment:
Not a blocker but just to bear in mind. How will this evolve over time? If we added other label to our configmaps we'd need to come here and add it also. I feel that the connection between this `resourceLabels` and the code that produces them is too far apart and only in our heads. It's a hard problem to solve :thinking:

Reply:
i believe is a hard problem to solve, but whenever we introduce a new resource type so check configmaps we should bear in mind, i guess the destroy will fail as there would be configmaps found for that resource